cloudberry/zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/TestQuery.scala

```diff
@@ -124,6 +124,17 @@ object TestQuery {
     aggregates = Seq(AggregateStatement(population, Sum, Field.as(Sum(population), "sum")))
   )
 
+  val TwitterDataSetForSQL = TwitterDataStoreForSQL.DatasetName
```

Comment:
I thought the TwitterDataSet was initialed in capitals, should I change it here?